src/DynamoCoreWpf/ViewModels/Watch3D/IWatch3DViewModel.cs

```diff
@@ -45,7 +45,18 @@ public struct RenderDescriptions
         /// </summary>
         public const string ManipulatorPlane  = "E75B2B0E31F1";
     }
-
+    /// <summary>
+    /// interface that defines properties of the Watch3dView required for preference serialization.
+    /// this interface should be merged with IWatch3dViewModel at Dynamo 2.0
+    /// </summary>
+    public interface IWatchPreferenceProperties
+    {
+        /// <summary>
+        /// Represents the name of current IWatch3DViewModel which will be saved in preference settings
+        /// </summary>
+        string PreferenceWatchName { get; }
```

Comment:
I understand only Watch3DViewModel is utilizing this property for now but from the name and purpose, I think we could move it one level up to IPreferences.cs and name it IPreferenceProperties. What do you think?